manifest.xml

```diff
@@ -23,12 +23,12 @@
        <depend_optional package="castxml" />
        <depend_optional package="gccxml" />
   -->
-  <depend package="boost" />
-  <depend package="facets" />
-  <depend package="libxml2" />
-  <depend package="pkg-config" />
-  <depend package="ruby-dev" />
-  <depend package="ruby-backports" />
+  <rosdep name="boost" />
+  <rosdep name="facets" />
+  <rosdep name="libxml2" />
+  <rosdep name="pkg-config" />
+  <rosdep name="ruby-dev" />
+  <rosdep name="ruby-backports" />
```

Comment:
This is required to find the matching keys in the [rosdep database](https://github.com/ros/rosdistro/tree/master/rosdep).
Alternatively the file could be renamed to `autoproj.xml` in the future, if that is an acceptable path forward for you.